org.eclipse.xtext.tests/src/org/eclipse/xtext/xtext/generator/ValidatorFragment2Tests.xtend

```diff
@@ -172,6 +172,41 @@ class ValidatorFragment2Tests extends AbstractGeneratorFragmentTests {
 			
 				@Override
 				protected void initialize(IAcceptor<PreferenceKey> acceptor) {
+					super.initialize(acceptor);
+					acceptor.accept(create(DEPRECATED_MODEL_PART, SeverityConverter.SEVERITY_WARNING));
+				}
+			}
+		'''.toString, concatenationClientToString(fragment.generateIssueProvider))
+	}
+	
+	@Test
+	def testGenerateConfigurableIssueProvideXbaser() {
```

Comment:
a very small cosmetic remark: I guess it should be "ProviderXbase" instead of "ProvideXbaser"

Reply:
Sure. Will correct.